tests/test_blade.py

```diff
@@ -5,7 +5,8 @@
 import os
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
-from OCC.TopoDS import TopoDS_Shape
+from OCC.Core.TopoDS import TopoDS_Shape, TopoDS_Solid
+from OCC.Extend.DataExchange import write_stl_file
```

Comment:
I think you can remove this line

Reply:
Yes, I have checked and you are right: the write_stl_file import is not needed. I will remove it, thanks.